### DISCUSSION TITLE
Toggle Layouts

### DISCUSSION THREAD

**Discussion by System-x64** [hyprwm/Hyprland#943] [Q&A] [answered]
Just a small doubt....is it pozzible to set a keybinding to toggle between master and dwindle? If so how?

**Comment by NotAShelf** [comment, marked as answer]
You can use  `hyprctl keyword general:layout dwindle`  or `hyprctl keyword general:layout master` to set the current layout.

Make a script that uses `hyprctl getoption` to get the current layout, and set the layout accordingly. Then bind the script to a key. Voila!

**Reply by System-x64** [reply]
but still...won't it be nice to just have this dispatcher inbuilt by default?

**Reply by NotAShelf** [reply]
That is something you should discuss with vaxry, not me. And that would require the dispatcher to be updated if there were to be new layouts added in the future. Eitherway, it's doable with bash.

**Reply by System-x64** [reply]
yea
